lib/ansible/modules/windows/win_description.ps1

```diff
@@ -0,0 +1,114 @@
+#!powershell
+
+# Copyright: (c) 2019, RusoSova
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#AnsibleRequires -CSharpUtil Ansible.Basic
+#AnsibleRequires -OSVersion 6.2
```

Comment:
What's the reason for this limitation, I would have assumed the registry structure has been the same for a long time and `Win32_OperatingSystem` has been around since Vista.